sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -9,12 +9,72 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     this.listenTo(this.categories, 'sync change:unread_count', this.render);
   },
 
+  events: {
+    'click #login-submit': 'loginSubmit',
```

Comment:
The submit happens regardless; what prevents it is the e.preventDefault(). If you make this ```submit #login```, it should still work and you won't need the other IDs in the tpl file.